CONTRIBUTING.md

```diff
@@ -1,275 +1,59 @@
-[case studies](case_studies/).
-
-1. Go to [Cloud9](https://c9.io/) and open your "hackclub" workspace.
-2. Make sure that the email address associated with git is correct.
-
-        $ git config user.email
-
-   If the email shown does not **exactly** match the email you used to sign up
-   to GitHub, reset it.
-
-        $ git config --global user.email "your@email.com"
-
-3. If you're not already on the `master` branch, switch to it.
-
-        $ git checkout master
-
-4. Update your Hack Club fork.
-
-        $ git pull https://github.com/hackclub/hackclub.git
-
-5. Create a new branch to add the case study to. Branch names should briefly
-   describe the change being made. They should be lowercase and use dashes (`-`)
-   instead of spaces. Since I'm adding the planning for our first meeting, I'm
-   going to call this branch `add-first-meeting-planning`. We call this a
-   feature branch.
-
-        $ git checkout -b add-first-meeting-planning
-
-6. Open up the file you want to modify for the case study and make any
-   additions/modifications. In this case, I'm going to open
-   `case_studies/lowell_high_school/2015-09-19_first_meeting/planning.md` and
-   add some example planning to it (create a folder with a corresponding format
-   if there isn't one already).
-
-    ![Adding some planning](https://i.imgur.com/HbM24cp.gif)
-
-7. Tell git to add all of your changes (make sure you've saved all of your
-   files).
-
-        $ git add --all
-
-8. Let's tell git to make a commit with our changes and attach a message to the
-   commit. Your message should reflect the change being made. In this case,
-   we're going to set the message to "Add Lowell first meeting planning." Change
-   the message appropriately.
-
-        $ git commit -m "Add Lowell first meeting planning."
-
-9. Push our branch with our changes to GitHub. Replace
-   `add-first-meeting-planning` with your branch name. Git will prompt you for
-   your GitHub username and password. When you start typing your password, it'll
-   look like nothing is happening -- don't worry, git is getting all of your
-   keystrokes and is hiding them for your privacy. Just hit enter when you're
-   done.
-
-        $ git push -u origin add-first-meeting-planning
-
-10. Go to your `hackclub` fork on GitHub (it's at
-    `https://github.com/{username}/hackclub`, mine is at
-    https://github.com/zachlatta/hackclub). You should see the following:
-
-    ![Recently pushed branches notification](https://i.imgur.com/NOeQyBe.png)
-
-11. Click the green "Compare & pull request" button to create a new pull
-    request. If appropriate, modify the title of the pull request and further
-    describe your changes in the description. Once you're ready, click "Create a
-    pull request".
-12. And you're done! Not too bad, huh? The pull request I created is over at
-    https://github.com/hackclub/hackclub/pull/163.
+![img/dance.gif](img/dance.gif)
```

Comment:
I know this may seem nit-picky but I think it's important. This gif is all white men, can we try to find a gif that has an equal gender ratio + bonus points for getting non white people in this as well. Why is this important? If I were a female and I were to see this, I would feel like this is not a place for me, not feel welcome, and not likely not contribute let alone participate in Hack Club.